src/core/dev/rfs_rule.cpp

```diff
@@ -88,6 +99,41 @@ bool rfs_rule::create(dpcp::match_params &match_value, dpcp::match_params &match
                *reinterpret_cast<uint64_t *>(match_value.dst.ipv6),
                *reinterpret_cast<uint64_t *>(match_value.dst_mac));
 
+    if (0) { // TEMP: Switch to 1 to enable DOCA Flow
```

Comment:
you had a param for that before, don't you want it here?

Reply:
This param will be added in RX path PR. Here it's just to be able to turn on for debugging. RX data path is not working yet. Only receiving a few packets.